### DISCUSSION TITLE
Does the network situation affect neovim's performances, especially for this plugin and dashboard.nvim?

### DISCUSSION THREAD

**Discussion by omegahh** [nvimdev/lspsaga.nvim#1016] [Q&A]
Yesterday, I was downloading a huge file from internet. The downloader almost occupied the full speed of the network card (about 40MB/s). Then I found serious lagging of dashboard.nvim when starting up the neovim and some Lspsaga functions like `gD` has no response until several minutes passed.

I think the I/O is enough because my system runs on HDD. So would the network occupation status affects the related performance?

**Comment by glepnir** [comment]
i don't think so ? performance of lspsaga is the language server performance . if it can return response fast . lspsaga show it will fast.

**Reply by omegahh** [reply]
OK, I will try to reproduce it by downloading the large file again, when I have some time for it.
